draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -239,31 +237,50 @@ The detailed algorithm is described in (#verifier_verification).
 
 This section defines data formats for SD-JWTs, Disclosures, Holder Binding JWTs and formats for combining these elements for transport.
 
-## Format of an SD-JWT
+## Format of an SD-JWT {#creating_sd_jwt}
 
-An SD-JWT is a JWT that MUST be signed using the Issuer's private key. The
-payload of an SD-JWT MUST contain the `_sd_alg` claim
-described in (#hash_function_claim). The SD-JWT payload MAY contain one or more selectively disclosable claims. It MAY also contain a Holder's public key or a reference
-thereto, as well as further claims such as `iss`, `iat`, etc. as defined or
-required by the application using SD-JWTs.
+An SD-JWT is a JWT that MUST be signed using the Issuer's private key.
+It MUST use a JWS asymmetric digital signature algorithm. It
+MUST NOT use `none` or an identifier for a symmetric algorithm (MAC).
+
+The payload of an SD-JWT is a JSON object according to the following rules:
+
+ 1. The payload MAY contain the `_sd_alg` key described in (#hash_function_claim).
```

Comment:
```suggestion
 1. The payload MAY contain the hash algorithm used by the Issuer to generate the digests as described in (#hash_function_claim).
```
because points 2 and 3 do not mention `_sd` and `...` (which I think is correct), for consistency, suggest not to mention `_sd_alg` too. otherwise my brain went like "why is `_sd` not metioned!"

Reply:
I don't think that this is needed. `_sd_alg` is very concise and concrete, so we should mention it. For `...` and `_sd` we say how exactly they are embedded further below (because it needs much more explanation than `_sd_alg`).

Bonus nitpick: The payload doesn't contain the hash algorith, but the hash algorithm's identifier.

Reply:
ok, that's fine.